### DISCUSSION TITLE
UTM not letting me go over 3072MiB of Ram

### DISCUSSION THREAD

**Discussion by stossy11** [utmapp/UTM#4765] [UTM Help (iOS)]
**Describe the issue**
When i try to run Any VM with more than 3072 MiB of ram it says"QEMU exited from an error: qemu-system cannot set up guest memory 'pc.ram' Cannot allocate memory

**Configuration**
* UTM Version: 4.0
* OS Version: 16.1
* Device Model: iPad Air 5 8GB of ram
* Is it jailbroken (name jailbreak used)? unJailbroken
* How did you install UTM? Using Altserver and also with jitstreamer

**Debug log**
[debug.log](https://github.com/utmapp/UTM/files/10164253/debug.log)

**Upload VM**
Dont Need to for this error 

**How To Reproduce (Extra)**
all you have to do to reproduce this error is to set the VMs Ram to 4096MiB of ram and try to start it

**Comment by osy** [comment]
That’s iOS’s limitation. Use a jailbreak if you want to bypass it.

**Reply by stossy11** [reply]
i found another way to bypass it by turning down the Jit Cache

**Reply by WindowsHome645** [reply]
Like how man?
